src/main/webapp/app/shared/orion/orion-connector.service.ts

```diff
@@ -215,7 +215,31 @@ export class OrionConnectorService implements ArtemisOrionConnector {
         theWindow().orionVCSConnector.selectRepository(repository);
     }
 
+    /**
+     * Orders the plugin to run the maven test command locally
+     */
     buildAndTestLocally(): void {
         theWindow().orionBuildConnector.buildAndTestLocally();
     }
+
+    /**
+     * Assess the exercise as Tutor. Triggers downloading/opening the exercise as tutor
```

Comment:
A small thing, but it would be a good idea to stay consistent with the capitalization of "Tutor/tutor" within the comment.